TestAssets/NonRestoredTestProjects/DotnetAddP2PProjects/App/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using NSLib1;
+using NSLib3;
+
+namespace ConsoleApplication
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+#if NETCOREAPP1_0
+            Lib1.HelloNCA();
+            Lib3.HelloNCA();
+#endif
+
+#if NET451
+            Lib1.HelloNet451();
+            Lib3.HelloNet451();
+#endif
+        }
+    }
+}
```

Comment:
These test projects seem quite complicated relative to what they're actually being used for in tests. Unless the complexity will add value through some test cases, I'd be very curious to actually test this with `dotnet new` and `dotnet new -t lib` so we see our e2e working well.